modules/perception/camera/test/camera_lib_lane_postprocessor_darkscnn_lane_postprocessor_test.cc

```diff
@@ -169,8 +169,8 @@ TEST(darkSCNNLanePostprocessor, camera_lane_postprocessor_point_test) {
   double yaw_adj = 0;
   double roll_adj = 0;
   std::string visual_camera = "onsemi_obstacle";
-  std::map<std::string, Eigen::Matrix4d> extrinsic_map;
-  std::map<std::string, Eigen::Matrix3f> intrinsic_map;
+  EigenMap<std::string, Eigen::Matrix4d> extrinsic_map;
```

Comment:
@emankov please comment.

Reply:
1. Camera tests were excluded from building for a long time. The following change was our try to reanimate camera tests, and then run them against both AMD and NVIDIA GPUs for correctness comparison. We didn't enable camera tests.
```
SHA-1: 9a5676655e12cb505f829cb949ec712cd1c4ddfb
[NVIDIA][tests][perception/camera][fix] Reanimate camera tests

+ Fix camera BUILD
+ Fix OpenCV issues
+ Fix Eigen / STL types mismatch
+ Fix warnings with signed/unsigned comparison

[Results]
+ 22 of 26 tests PASS
+ 4 of 26 tests FAILED (logs attached):
  camera_app_obstacle_camera_perception_test
  camera_common_undistortion_handler_test
  camera_lib_lane_detector_darkscnn_lane_detector_test
  camera_lib_lane_postprocessor_darkscnn_lane_postprocessor_test

[ToDo]
+ Fix camera BUILD, run and fix (if possible) camera tests on AMD GPU
+ Play with still-commented camera tests
```
2. `EigenMap` is expected in the test's code, otherwise compilation error.